src/components/Layout/index.module.css

```diff
@@ -24,3 +31,22 @@
     margin-bottom: 2rem;
   }
 }
+
+.footer {
+  display: flex;
+  padding: 5rem 2rem 2rem;
+  color: #fff;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+
+  :is(a) {
+    color: #fff;
+    text-decoration: none;
+    font-weight: 700;
+    margin-left: 0.25rem;
+    &:hover {
+      text-decoration: underline;
+    }
+  }
+}
```

Comment:
Can we simply align footer content to center. Looks weird to me on desktop.

![image](https://github.com/oasisprotocol/dapp-wrose/assets/9722540/fb03cc05-e591-4c11-9595-a221c7e5c805)

Reply:
Based on Don's feedback we will add playground links on the right side in the near future in the footer that's why we should align build info on left side.

Reply:
But this app isn't really a playground app. That's why it's hosted on a separate domain as well.

Reply:
indeed it's not. will chat with Don one more time